flex/bin/CMakeLists.txt

```diff
@@ -56,4 +49,14 @@ if(BUILD_HQPS)
         endif()
         # install the script
         install(PROGRAMS load_plan_and_gen.sh DESTINATION bin)
-endif()
\ No newline at end of file
+endif()
+
+find_package(Boost COMPONENTS program_options REQUIRED)
```

Comment:
Dependency to `boost program_options` has already been added in flex/CMakeLists.txt, no need to find again.